src/wristpy/processing/calibration.py

```diff
@@ -65,6 +66,9 @@ class Calibration:
     Attributes:
             chunked: If true will preform calibration on subsets of data if false will
                 calibrate on entire dataset.
+            method: Define the method used to find the calibration scaling and offset
+                that minimizes the calibration error. Current options are:
+                    'GGIR', or the default 'Gradient'.
```

Comment:
Small nit re the terminology, `scipy.minimize` doesn't technically use gradient descent by default, but rather LBFGS (a better algorithm in this context). Might consider calling the method `'minimize'` or `'scipy'` or sth.

Reply:
huh, I thought LBFGS was a gradient descent algorithm, TIL

Reply:
Tbf, it is definitely a nit comment 😅. It is true that LBFGS uses only first order information. But the step direction it computes is better than the naive negative gradient direction (using an approximate Hessian).